aws-android-sdk-core/build.gradle

```diff
@@ -20,14 +19,14 @@ android {
 }
 
 dependencies {
-    api 'com.google.code.gson:gson:2.2.4'
-    implementation 'com.fasterxml.jackson.core:jackson-core:2.1.1'
+    api 'com.google.code.gson:gson:2.8.6'
+    implementation 'com.fasterxml.jackson.core:jackson-core:2.11.0'
```

Comment:
ah I see what you meant hahaha. Where do we use jackson even?

Reply:
No clue. After this goes in, I'm going to come back and see if I can remove one of these.